etc/profile-m-z/nvim.profile

```diff
@@ -0,0 +1,39 @@
+caps.drop all                                                                                                                                                                                                        
+netfilter                                                                                                                                                                                                            
+nodbus                                                                                                                                                                                                               
+nodvd                                                                                                                                                                                                                
+nogroups                                                                                                                                                                                                             
+noinput                                                                                                                                                                                                              
+nonewprivs                                                                                                                                                                                                           
+noroot                                                                                                                                                                                                               
+notv                                                                                                                                                                                                                 
+nou2f                                                                                                                                                                                                                
+novideo                                                                                                                                                                                                              
+protocol unix,inet,inet6                                                                                                                                                                                             
+seccomp                                                                                                                                                                                                              
```

Comment:
can we use
- seccomp-block-secondary
- tracelog
- shell none
- noprinters
- no3d
- machine-id
- ipc-namespace
- net none
- disable-X11
- disable wayland
- disable runuser